honeybadger/payload.py

```diff
@@ -94,15 +95,42 @@ def create_payload(exception, exc_traceback=None, config=None, context={}):
     if exc_traceback is None:
         exc_traceback = sys.exc_info()[2]
 
-    payload = default_plugin_manager.generate_payload(config, context)
+    request_payload = default_plugin_manager.generate_payload(config, context)
 
-    return {
+    payload =  {
         'notifier': {
             'name': "Honeybadger for Python",
             'url': "https://github.com/honeybadger-io/honeybadger-python",
             'version': __version__
         },
         'error':  error_payload(exception, exc_traceback, config),
         'server': server_payload(config),
-        'request': payload,
+        'request': request_payload,
+        'details': {}
     }
+    
+    if config.is_aws_lambda_environment:
+
+        # Should this be in the lambda plugin instead?
+        # We can't manipulate the payload directly from the plugin
+        # So this makes sense for now
```

Comment:
I like this comment! :) I would *prefer* that this could be moved to the plugin, but I think the issue is that the plugin system doesn't currently support modifying the full payload, right? If that's the case, I think for this PR I would go with this as-is, but then let's create a new issue to modify the plugin system to support whatever is needed to move this into the lambda plugin later (as a refactor). What do you think of that idea?

Reply:
I totally agree. That will ensure complete separation of concerns.

Reply:
I totally agree. That will ensure complete separation of concerns.